lib/osf-components/addon/components/registries/my-registrations-list/drafts/component.ts

```diff
@@ -0,0 +1,10 @@
+import Component from '@glimmer/component';
+
+export default class MyRegistrationsDraftListComponent extends Component {
+    queryParam = {
+        embed: [
+            'provider',
+            'registration_schema',
```

Comment:
Should we embed bibliographic contributors here as well?

Reply:
Oh, never mind. I see what's happening; if you load the page with the draft tab active, it's grabbing the bibliographic contributors for the other tab.